### DISCUSSION TITLE
Phone in 'charging' status but battery keep being drain

### DISCUSSION THREAD

**Discussion by marcussacana** [VR-25/acc#146] [General]
Hi, I have an Redmi Note 9S, I want use this tool to extend my battery life by slowing down/disabling the fast charger feature, also preventing to being charged more than 80%...

I used this tool a time ago and worked fine, not sure if it was after a certain ACC magisk update or after I updated the system (I still in Android 10) but now my phone when ON say 'charing' but the battery keep being drain, then I want a bit of help since I don't understand much of hardware to be able to setup it...

I tried as well run this command `acc -e 80%`
and This is what I got: 
![image](https://user-images.githubusercontent.com/10576957/165140609-0a9c7f4b-00f6-49f4-afc8-c18f7c31bd07.png)
But even after he said has 'disabled' the battery don't charged at all, only when I turn off the phone or disable the ACC module to be able charge again.

I did something bad maybe?
The last setup command that I did trying solve the charing has this one:
`acc -s pc=80 rc=30 mcc=500 mcv=3700` (with a big fear of put high voltage or amp, since I don't know what is the default, lol)

**Comment by VR-25** [comment]
A voltage limit that low (3700 mV) won't let the battery charge to 80%. Voltage limits capacity. Restore the default voltage limit (acc -sv -) or increase it -- and see if the issue persists.

**Comment by marcussacana** [comment]
Fine, I executed this just now: 
![image](https://user-images.githubusercontent.com/10576957/165142521-0dccd134-b690-460d-8558-4380d4fda511.png)
Then I will wait a bit and check if charge, my battery is at 20% currently

**Comment by marcussacana** [comment]
for reference:
![image](https://user-images.githubusercontent.com/10576957/165142711-043c3367-7cca-4dc2-a08f-718ae77b32a5.png)
![image](https://user-images.githubusercontent.com/10576957/165142770-69724926-ba60-4599-90f1-c1f50f41b1be.png)
![image](https://user-images.githubusercontent.com/10576957/165142794-0b03fea1-c64b-4530-a20d-de9351de2aac.png)

**Comment by marcussacana** [comment]
After one hour, still at 20% :rofl:

**Comment by marcussacana** [comment]
Okay, now it changed to 19%

**Comment by VR-25** [comment]
Reboot.

**Comment by marcussacana** [comment]
Fine, rebooted, 18% now, let's see.

**Comment by marcussacana** [comment]
Yup, 28% now, I think it's better now.
